src/workloads/query/TelemetryQueryShapes.yml

```diff
@@ -0,0 +1,133 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of telemetry with different query shapes.
```

Comment:
This isn't entirely accurate anymore. Could you update it to say something more about how it was designed to stress the telemetry collection - but doesn't actually enable telemetry. It can be run with or without telemetry for comparisons.